pkg/sql/sem/builtins/builtins.go

```diff
@@ -7491,6 +7487,24 @@ expires until the statement bundle is collected`,
 		tree.FunctionProperties{
 			Category: builtinconstants.CategorySystemInfo,
 		},
+		tree.Overload{
+			Types: tree.ParamTypes{
+				{Name: "span", Typ: types.BytesArray},
+				{Name: "stripped", Typ: types.Bool},
+				// NB: The function can be called with an AOST clause that will be used
+				// as the `end_time` when issuing the ExportRequests for the purposes of
+				// fingerprinting.
+			},
+			ReturnType: tree.FixedReturnType(types.Int),
+			Fn: func(ctx context.Context, evalCtx *eval.Context, args tree.Datums) (tree.Datum, error) {
+				if skipTimestamp := *args[1].(*tree.DBool); !skipTimestamp {
+					return nil, errors.New("cannot use crdb_internal.fingerprint skip timestamp overload with skip_ts=false")
```

Comment:
yeah it would be equivalent. I suppose we could treat `SELECT crdb_internal.fingerprint(span, false)` as syntactic sugar for `SELECT crdb_internal.fingerprint(span, 0::TIMESTAMPTZ, false)`. 

I initially disallowed this form to prevent confusion-- "if you use this first overload, you can only use it for the stripped fingerprint".

I'll change this to allow the sugared syntax.